source/common/router/router.h

```diff
@@ -380,7 +381,8 @@ class Filter : Logger::Loggable<Logger::Id::router>,
   void onUpstreamMetadata(Http::MetadataMapPtr&& metadata_map);
   void onUpstreamComplete();
   void onUpstreamReset(UpstreamResetType type,
-                       const absl::optional<Http::StreamResetReason> reset_reason);
+                         const absl::optional<Http::StreamResetReason> &reset_reason,
```

Comment:
nit: no reason to pass by reference

Reply:
ah, bad merge, fixed. btw this shouldn't be const if passing by value though.